### PR TITLE
Add test documentation to the PR template

### PR DESCRIPTION
**Pull request by dpoeschl**
Only changes the PR template

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -31,3 +31,7 @@ How did we miss it?  What tests are we adding to guard against it in the future?
 **How was the bug found?**
 
 (E.g. customer reported it vs. ad hoc testing)
+
+**Test documentation updated?**
+
+If this is a new non-compiler feature, update https://github.com/dotnet/roslyn/wiki/Manual-Testing once you know which release it is targeting.		 +If this is a new non-compiler feature, or a significant improvement to an existing feature, update https://github.com/dotnet/roslyn/wiki/Manual-Testing once you know which release it is targeting.
```
Comment (Pilchie) [inline .github/PULL_REQUEST_TEMPLATE.md]: Copy/pasted twice?

**Comment by dpoeschl** [comment]
@Pilchie Fixed.

**Comment by dpoeschl** [comment]
retest windows_determinism_prtest please

**Comment by dpoeschl** [comment]
retest windows_release_vs-integration_prtest please

**Pull request merged**